src/types/homepage.ts

```diff
@@ -89,11 +89,16 @@ export type HomepageEditorHeroSection =
   | EditorHeroDropdownSection
   | EditorHeroHighlightsSection
 
+export type HeroFrontmatterSection = { hero: HomepageEditorHeroSection }
+export type ResourcesFrontmatterSection = { resources: Record<string, unknown> }
+
+export type EditorHomepageFrontmatterSection =
+  | HeroFrontmatterSection
+  | ResourcesFrontmatterSection
```

Comment:
Hmm should this also include the other types of sections (infobar, infopic)?

Reply:
yes - not done due to time. there's an existing jira issue filed for this (off the top of my head, IS-408 or IS-409)